uyuni/base/uyuni-base.spec

```diff
@@ -110,11 +110,8 @@ getent passwd %{apache_user} >/dev/null && %{_sbindir}/usermod -a -G susemanager
 %endif
 
 %files common
-%if 0%{?rhel} == 6
-%doc LICENSE
-%else
+%{!?_licensedir:%global license %doc}
```

Comment:
We tried this already, but it did not work on RHEL6. Please try there.

Reply:
Done. Works just fine. Links are at the description.

Maybe you tried something slightly different.